Makefile

```diff
@@ -68,7 +68,7 @@ FUZZER_BUILD ?= 0
 # NOTE: when changing any value below, you'll need to adjust WORKSPACE file by hand:
 #         - uncomment source build section, comment binary section
 #         - adjust binary version path - version variable is not passed to WORKSPACE file!
-OV_SOURCE_BRANCH ?= 6a8079bd6b536ea67e36782ae41cbd214b2acc53  # 29 May releases/2024/2
+OV_SOURCE_BRANCH ?= 5c0f38f83f62fdabcdc980fa6dc3ed1ea16c8a05  # 29 May releases/2024/2
```

Comment:
I don't think the date in the comment is correct. Could you update it or remove entirely?

Reply:
that is correct date from the ov build manifest